### PR TITLE
Law: Mayer's relation for homogeneous substances in general

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/thermodynamics/isochoric_and_isobaric_heat_capacities_of_homogeneous_substance.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## Mayer's relation is the relation between heat capacity at constant pressure and heat
+## capacity at constant volume.
+
+# Law: C_p - C_V = V * T * (alpha_V)**2 / beta_T
+## C_p - heat capacity at constant pressure
+## C_V - heat capacity at constant volume
+## V - volume
+## T - temperature
+## alpha_V - [thermal expansion coefficient](../../definitions/volumetric_coefficient_of_thermal_expansion.py)
+## beta_T - isothermal compressibility
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/isochoric_and_isobaric_heat_capacities_of_homogeneous_substance.py:22]: What's this?

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/thermodynamics/isochoric_and_isobaric_heat_capacities_of_homogeneous_substance.py (hunk 1 above)
Comment (alesanter) [reply symplyphysics/laws/thermodynamics/isochoric_and_isobaric_heat_capacities_of_homogeneous_substance.py:22]: I think it is easier to create the definition file and give a link to it here

Comment (blackyblack) [reply symplyphysics/laws/thermodynamics/isochoric_and_isobaric_heat_capacities_of_homogeneous_substance.py:22]: Ok. Put TODO comment, in this case.

Comment (alesanter) [reply symplyphysics/laws/thermodynamics/isochoric_and_isobaric_heat_capacities_of_homogeneous_substance.py:22]: Opened #586

Comment (blackyblack) [reply symplyphysics/laws/thermodynamics/isochoric_and_isobaric_heat_capacities_of_homogeneous_substance.py:22]: Merged. You can merge with it and put a link to this parameter definition.

Comment (alesanter) [reply symplyphysics/laws/thermodynamics/isochoric_and_isobaric_heat_capacities_of_homogeneous_substance.py:22]: Yes, that was my intention

**Pull request merged**